registry/src/store/reducers/uploadReducer.js

```diff
@@ -0,0 +1,35 @@
+import {
+  UPLOAD_PACKAGE,
+  UPLOAD_PACKAGE_SUCCESS,
+  UPLOAD_PACKAGE_ERROR,
+} from "../actions/uploadActions";
+
+const initialState = {
```

Comment:
There is data field here but don't see it's usage inside the reducer. There is no message field here. I think to avoid the confusion it can renamed as ``error`` if it's the error message and ``success`` if it's the success message. 

But, if you have a better idea, you should implement it.

Reply:
This is also fixed. 👍